test/jdk/sun/security/pkcs11/ec/TestECDSA.java

```diff
@@ -167,6 +180,17 @@ public void main(Provider provider) throws Exception {
         }
         test(provider, pub521, priv521, sig521);
 
+        // This test is known to be executed in two ways:
+        // 1. Direct execution of this test for testing sun.security.pkcs11
+        // functionality. Skip brainpoolP512r1 tests in this case since the PKCS11
+        // provider does not support them.
+        // 2. Running the testcase in sun/security/ec. Expect brainpoolP512r1
+        // curve to be present execute the brainpoolP512r1 specific test.
```

Comment:
Something is missing in this comment: perhaps you meant "present _**and**_ execute"?

Reply:
Changed from `present execute` to `present and execute`